src/site/asciidoc/manual/configuration.adoc

```diff
@@ -1146,6 +1146,13 @@ included, such as ``${main:\--file:-app.properties}`. This would use the
 `MainMapLookup` for a key named `--file`. If the key is not found then
 <code>app.properties</code> would be used as the default value.
 
+[#DisablesMessagePatternLookups]
+== Disables Message Pattern Lookups
+Message is processed (by default) by lookups, for example if you defined
```

Comment:
Message is processed (by default)
->
A message is processed

Reply:
Actually it might be simpler for a native English speaker to rewrite the whole thing instead of editing here which is awkward.

Reply:
@garydgregory You could merge this PR then polish it.